src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -195,18 +200,14 @@ full name
 
         private readonly Logger _logger;
         private readonly string _name;
-        private static readonly Slice AllTombstonesEtagsSlice;
-        private static readonly Slice LastEtagSlice;
 
         // this is only modified by write transactions under lock
         // no need to use thread safe ops
         private long _lastEtag;
 
         private readonly StringBuilder _keyBuilder = new StringBuilder();
 
-        public string DataDirectory;
         public DocumentsContextPool ContextPool;
-        private UnmanagedBuffersPoolWithLowMemoryHandling _unmanagedBuffersPool;
```

Comment:
_unmanagedBuffersPool no one is using this. As I went over this code I made there a few improvements. We should convert all strings there so using Slices.